datafusion/core/tests/sqllogictests/src/engines/datafusion/mod.rs

```diff
@@ -18,7 +18,7 @@
 use std::path::PathBuf;
 use std::time::Duration;
 
-use crate::output::{DFColumnType, DFOutput};
+use crate::engines::output::{DFColumnType, DFOutput};
```

Comment:
I moved the `output.rs` into the `engines/` folder. I just prefer narrowing the scope of files to where they are used, and there's no need to expose the type outside of the `engines/` folder now.